.circleci/config.yml

```diff
@@ -1,3 +1,20 @@
+aliases:
+- &docs-job
+  docker:
+  - image: cloudreach/sceptre-circleci:0.3
+    environment:
+      CONTAINER_DEST_REPO_DIR: "/home/circleci/docs"
+      DEST_REPO: "https://github.com/sceptre/docs.git"
+  steps:
```

Comment:
Probably worth adding a save_cache/restore_cache step here

Reply:
Right I could probably cache `CONTAINER_DEST_REPO_DIR`

however this would require me to update `github-pages.sh` to run different `git` operations based on cache. Maybe I could do it as a next step after we merge.

Reply:
Yes ok - can you write up a ticket for adding caching to build steps?